src/scss/tippy/_base-tooltips.scss

```diff
@@ -0,0 +1,28 @@
+@import 'popper-theme-arrows';
+
+/**
+ * Tippy tooltips have the following base structure:
+ *
+ * <div class="tippy-popper" x-placement="top">
+ *   <div class="tippy-tooltip">
+ *     <div class="tippy-content">
+ *       [[ Content Here ]]
+ *     </div>
+ *   </div>
+ * </div>
+ */
+
+.tippy-popper {
+  @include popper-theme-arrows($arrow-size: $arrow-size, $arrow-pointer-events: $arrow-pointer-events);
```

Comment:
looks like they go here @rwwagner90

Reply:
Yep, I noted that [here].(https://github.com/shipshapecode/shepherd/pull/261#discussion_r223237023)

I think there's a future story we need to make for figuring out exactly how we're offering custom theming (which Tippy could be able to help with, too!)

Reply:
👍